### PR TITLE
chore(react-chart): update demos navigation

### PR DESCRIPTION
**Pull request by DmitryBogomolov**
*Stacked Area* demo is split into *Stacked Area* and *Fullstacked Area*.

### CODE REVIEW

File: site/_data/demos/navigation.yml [hunk 1]
```diff
@@ -28,7 +28,9 @@
         path: /react/chart/demos/area/spline-area
       - title: Stacked Area
         path: /react/chart/demos/area/stacked-area
-      - title: Step Area
+      - title: Fullstacked Area
+        path: /react/chart/demos/area/fullstacked-area
+      - title: Step Areacd
```
Comment (churkin) [inline site/_data/demos/navigation.yml]: typo?

File: site/_data/demos/navigation.yml [hunk 2]
```diff
@@ -28,7 +28,9 @@
         path: /react/chart/demos/area/spline-area
       - title: Stacked Area
         path: /react/chart/demos/area/stacked-area
-      - title: Step Area
+      - title: Fullstacked Area
```
Comment (churkin) [inline site/_data/demos/navigation.yml]: `Full Stack Area` or `Fully Stacked Area` :)

File: site/_data/demos/navigation.yml (hunk 1 above)
Comment (DmitryBogomolov) [reply site/_data/demos/navigation.yml]: yes

File: packages/dx-react-chart/demos/area/fullstacked-area.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# React Chart Full-Stacked Area
+
+The demo shows how to create Full-Stacked Area Series using the [Stack](../../docs/reference/stack.md) plugin.
```
Comment (RomanTsukanov) [inline packages/dx-react-chart/demos/area/fullstacked-area.md]:
```suggestion
This demo shows how to use the [Stack](../../docs/reference/stack.md) plugin to create a Full-Stacked Area series.
```

File: packages/dx-react-chart/demos/area/stacked-area.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # React Chart Stacked Area
 
-The demo shows how to create Stacked and Fullstacked Area Series using the [Stack](../../docs/reference/stack.md) plugin.
+The demo shows how to create Stacked Area Series using the [Stack](../../docs/reference/stack.md) plugin.
```
Comment (RomanTsukanov) [inline packages/dx-react-chart/demos/area/stacked-area.md]:
```suggestion
This demo shows how to use the [Stack](../../docs/reference/stack.md) plugin to create a Stacked Area series.
```

File: packages/dx-react-chart/demos/bar/fullstacked-bar.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
-# React Chart Fullstacked Bar
+# React Chart Full-Stacked Bar
 
-The demo shows how to use the [Stack](../../docs/reference/stack.md) plugin to create Fullstacked Bar Series.
+The demo shows how to use the [Stack](../../docs/reference/stack.md) plugin to create Full-Stacked Bar Series.
```
Comment (RomanTsukanov) [inline packages/dx-react-chart/demos/bar/fullstacked-bar.md]:
```suggestion
This demo shows how to use the [Stack](../../docs/reference/stack.md) plugin to create a Full-Stacked Bar series.
```

**Pull request merged**